internal/cmd/billing/command_cost_list.go

```diff
@@ -0,0 +1,114 @@
+package billing
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+type commandList struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+type costOut struct {
+	StartDate           string `human:"Start Date" serialized:"start_date"`
+	EndDate             string `human:"End Date" serialized:"end_date"`
+	Granularity         string `human:"Granularity" serialized:"granularity"`
```

Comment:
Do you mind writing some integration tests? I'd love to see an example of what this output looks like. Take a look at the `test/` directory!